### PR TITLE
Mmca 3326

### PR DESCRIPTION
**Pull request by derekspalmer**

### CODE REVIEW

File: app/views/DutyDefermentRequestedStatements.scala.html [hunk 1]
```diff
@@ -24,7 +24,11 @@
 @this(
     layout: Layout,
     missing_documents_guidance: components.missing_documents_guidance,
-    duty_deferment_file: components.duty_deferment_file
+    duty_deferment_file: components.duty_deferment_file,
+    h1: components.h1,
+    h2: components.h2,
+    p: components.p,
+    p2: components.p2
```
Comment (zachhall1234) [inline app/views/DutyDefermentRequestedStatements.scala.html:31]: Not sure I'm being blind but can't see where this p2 is being used.

File: app/views/components/p2.scala.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+@*
```
Comment (zachhall1234) [inline app/views/components/p2.scala.html:1]: Not too sure if I'm missing the reasoning behind this but looks like it matches the p.scala.html component code.

**Review by zachhall1234 — APPROVED** [review]

File: app/views/DutyDefermentRequestedStatements.scala.html (hunk 1 above)
Comment (derekspalmer) [reply app/views/DutyDefermentRequestedStatements.scala.html:31]: Yep - needs removing.

File: app/views/components/p2.scala.html (hunk 2 above)
Comment (derekspalmer) [reply app/views/components/p2.scala.html:1]: Nope - you're correct. 
I was thinking of introducing a new element for one of the divs but it got too complicated. Forgot to remove it!

**Pull request merged**